### PR TITLE
Ubuntu python35

### PR DESCRIPTION
**Pull request by nastasi-oq**
With this PR we move to use oq-python3.5 infrastructure to build and package ubuntu.
Build on LaunchPad: https://launchpad.net/~nastasi-oq/+archive/ubuntu/test/+build/14354799

### CODE REVIEW

File: helpers/whlsetup.sh [hunk 1]
```diff
@@ -69,6 +69,7 @@ while [ $# -gt 0 ]; do
             shift
             ;;
```
Comment (daniviga) [inline helpers/whlsetup.sh]: Please remove all the code block related to `-2` and just make `-3` the default.

File: helpers/whlsetup.sh [hunk 2]
```diff
@@ -123,37 +124,9 @@ if [ "$USE_PIP" == "true" ]; then
...
-    pip install ${nodeps} --no-index --prefix ${DEST} ${SRC[@]}
+    pip3 install ${nodeps} --no-index --prefix ${DEST} ${SRC[@]}
 
     # Cleanup
-    deactivate
-    rm -Rf $VENV
-
-    # replace scripts hashbang with the python executable provided
-    # by the system, instead of the one provided by virtualenv
-    find ${DEST} -type f -print0 | xargs -0 sed -i "s|${VENV}/bin/python.*|$python|g"
```
Comment (daniviga) [inline helpers/whlsetup.sh]: This is still needed. Please keep it updating just the comment.

File: debian/control [hunk 3]
```diff
@@ -1,23 +1,43 @@
-Source: python-oq-libs
+Source: python3-oq-libs
 Section: python
 Priority: extra
 Maintainer: Matteo Nastasi (GEM Foundation) <nastasi@openquake.org>
-Build-Depends: debhelper (>= 7.0.50~), lsb-release, curl, python (>=2.7), virtualenv, python-virtualenv, python-wheel
+Build-Depends: debhelper (>= 7.0.50~), lsb-release, curl, python-wheel, oq-python3.5
```
Comment (daniviga) [inline debian/control]: Why `python-wheel` ?

File: helpers/whlsetup.sh (hunk 2 above)
Comment (nastasi-oq) [reply helpers/whlsetup.sh]: VENV not exists, what you mean ?

Comment (daniviga) [reply helpers/whlsetup.sh]: You're right!

File: debian/changelog [hunk 4]
```diff
@@ -1,7 +1,14 @@
+python3-oq-libs (1.5.0) stable; urgency=low
```
Comment (daniviga) [inline debian/changelog]: LGTM but fix the version

**Pull request merged**